### PR TITLE
Change "Remove Wheel" Action's Color Based on Damage

### PR DESCRIPTION
**Pull request by 654wak654**
**When merged this pull request will:**
- Title.
- Sometimes I have to remove all 6 wheels of the Otokar Arma to find which 2 were effected by an RPG hit 😢.

Screenshots: http://imgur.com/a/N1SgY

### CODE REVIEW

File: addons/repair/functions/fnc_modifySelectionInteraction.sqf [hunk 1]
```diff
@@ -0,0 +1,32 @@
+/*
+ * Author: 654wak654
+ * Modifies interaction color of vehicle selection to show it's current damage
```
Comment (jonpas) [inline addons/repair/functions/fnc_modifySelectionInteraction.sqf]: Missing final dot. `its current damage`.

**Comment by 654wak654** [comment]
> I am assuming you used same colors as for wheels themselves?
Yup, exact same color as the damage_X stuff

**Comment by jonpas** [comment]
Do other repair points use similar levels of damage that this could be added to? I am thinking all points could use this in that case.

**Comment by 654wak654** [comment]
> Do other repair points use similar levels of damage that this could be added to? I am thinking all points could use this in that case.

Thought of the same thing but it's less useful with other hitpoints, since their action doesn't show up unless they're damaged anyway.

**Comment by jonpas** [comment]
You are right, little point since they show up only when damaged, additionally their hitpoints are shown in when you sit in the vehicle.

**Pull request merged**